src/dss/main.py

```diff
@@ -30,5 +31,37 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+@main.command(name="logs")
+@click.option(
+    "--parts",
+    type=click.Choice(["notebooks", "mlflow"]),
+    required=True,
+    help="Specify which part's logs you want to retrieve: 'notebooks' or 'mlflow'.",
```

Comment:
I was having a hard time understanding what `part` means. Would it make sense to have something like `dss logs <mlflow|notebooks> <--name=NAME>`? And name can be an option exclusively for Notebooks so you don't have to do the `Warning: The 'name' parameter will be ignored for 'mlflow' logs.` workaround.